exareme2/worker/Dockerfile

```diff
@@ -52,5 +53,5 @@ RUN chmod 775 /home/bootstrap.sh
 ENV CREDENTIALS_CONFIG_FOLDER=/opt/credentials
 RUN mkdir $CREDENTIALS_CONFIG_FOLDER
 VOLUME $CREDENTIALS_CONFIG_FOLDER
-
+ENV PYTHONPATH=$CODE_PATH
```

Comment:
Add this line higher up, below the ENV PATH

The reason for that is that this line won't need to be re-build when there is a change in the lines above.